open_instruct/mix_data.py

```diff
@@ -14,12 +14,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from open_instruct.finetune import FlatArguments
+
 # script for mixing and saving data
-from .utils import ArgumentParserPlus, FlatArguments, get_datasets
+from open_instruct.utils import ArgumentParserPlus, get_datasets
```

Comment:
Something that was missed in a prior update?

Reply:
It's in the OLMo Again PR, but yes it should've been updated when @vwxyzjn refactored args